src/testing/unit/test_json_helper.py

```diff
@@ -24,3 +29,47 @@ def test_write_json_file():
     assert expected == actual
 
     remove(file_path)
+
+
+def test_read_json_file():
+    """
+    Tests the read_json_file function.
+    """
+    file_path = "src/testing/test.json"
```

Comment:
Sonar has code smells issues with this, feel free to fix this but I am also keen to fix it as it is my code. Let me know.